### PR TITLE
Update default controls in README.txt

### PR DESCRIPTION
**Pull request by Wuzzy2**
The controls mentioned in `README.txt` are pretty outdated and sometimes even wrong or misleading.
This PR updates this section.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -27,39 +27,47 @@ This game is not finished
...
+- Right mouse button: Build/use
+- Shift + right mouse button: Build (without using)
+- I: Inventory menu
+- Mouse wheel: Select item
+- 0-9: Select item
+- Z: Zoom (needs zoom privilege)
+- T: Chat
+- /: Commad
+
+- Esc: Pause menu/abort/exit (pauses only singleplayer game)
```
Comment (paramat) [inline README.txt]: Esc doesn't cause abort or exit directly, so i think 'Pause and access exit menu'.

File: README.txt [hunk 2]
```diff
@@ -27,39 +27,47 @@ This game is not finished
...
-- Esc: pause menu (pauses only singleplayer game)
-- R: Enable/Disable full range view
+- Move mouse: Look around
+- W, A, S, D: Move
+- Space: Jump/move up
+- Shift: Sneak/move down
+- Q: Drop itemstack
+- Shift + Q: Drop single item
+- Left mouse button: Mine/punch/take item
+- Right mouse button: Build/use
```
Comment (paramat) [inline README.txt]: 'Place' is better, 'building water/flower etc' sounds weird.

**Comment by paramat** [comment]
Line comments, otherwise good.

File: README.txt (hunk 1 above)
Comment (Wuzzy2) [inline README.txt]: Yes, it does. Pressing Esc in main menu exits Minetest. In fullscreen mode this is even the only way to exit Minetest from the main menu directly.

File: README.txt [hunk 3]
```diff
@@ -27,39 +27,47 @@ This game is not finished
...
-
-- Esc: pause menu (pauses only singleplayer game)
-- R: Enable/Disable full range view
+- Move mouse: Look around
+- W, A, S, D: Move
+- Space: Jump/move up
+- Shift: Sneak/move down
+- Q: Drop itemstack
+- Shift + Q: Drop single item
+- Left mouse button: Mine/punch/take item
```
Comment (Wuzzy2) [inline README.txt]: OK.

File: README.txt (hunk 2 above)
Comment (Wuzzy2) [inline README.txt]: OK.

File: README.txt (hunk 1 above)
Comment (paramat) [inline README.txt]: I was referring to 'exiting to OS', but i can see this can mean exiting world.
Ok no problem then.

Comment (paramat) [inline README.txt]: I see.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
c0e56c6011fa6252a33894c560f8a16445466b8b

**Pull request closed without merging**